src/rules/isort/rules/organize_imports.rs

```diff
@@ -43,6 +44,17 @@ fn extract_indentation_range(body: &[&Stmt]) -> Range {
     Range::new(Location::new(location.row(), 0), location)
 }
 
+/// Compares two strings, returning true if they are equal modulo whitespace
+/// at the start of each line.
+fn matches_ignoring_indentation(val1: &str, val2: &str) -> bool {
```

Comment:
This helper might be useful elsewhere, although I wasn't sure what the best place for it would be.